src/templates/pix-download.html.php

```diff
@@ -33,8 +33,10 @@
                             <div>
                                 <?php
                                     $now = new DateTime('now', new DateTimeZone('America/Sao_Paulo'));
-                                    $pix_epiration = new DateTime($subscription['bill']['pix_expiration']);
-
+                                    $pix_epiration = new DateTime(
```

Comment:
🎨  Acho que aqui faltou um `x`, ou era assim mesmo?
```suggestion
                                    $pix_expiration = new DateTime(
```

Reply:
@KarinaSakata Realmente faltou. Corrigido.